tempesta_fw/http_frame.c

```diff
@@ -943,8 +946,10 @@ tfw_h2_settings_ack_process(TfwH2Ctx *ctx)
 {
 	T_DBG3("%s: parsed, stream_id=%u, flags=%hhu\n", __func__,
 	       ctx->hdr.stream_id, ctx->hdr.flags);
+
+	ctx->hpack.max_window = ctx->lsettings.hdr_tbl_sz;
 	/*
-	 * TODO: apply settings ACK.
+	 * TODO: apply other local settings on ACK receiving.
 	 */
 }
 
```

Comment:
1. The most significant bit (so called "reserved bit") must be ignored when receiving [RFC 7540 section 4.1](https://tools.ietf.org/html/rfc7540#section-4.1). It is discarded in `tfw_h2_unpack_frame_header()`.
2. The idea behind this macro is to process three different cases: a) continue processing in caller procedure; b) return with T_OK value to silently ignore the incoming frame and/or drop current stream; c) return with error value (T_DROP or others) to stop processing and drop the current connection. In variant with separate function instead of this macro - additional, rather complicated logic around of each call to this function will be needed, to test return value and choose the one of three further processing flows specified above.